assets/src/styles/blocks/core-overrides/Image.scss

```diff
@@ -2,20 +2,20 @@
   figure,
   img {
     border-radius: 50%;
-    height: $size;
-    width: $size;
+    max-height: $size;
   }
 
   img {
     object-fit: cover;
+    aspect-ratio: 1;
   }
 }
 
 @mixin square-image-size($size) {
   figure,
   img {
-    height: #{$size};
-    width: #{$size};
+    height: $size;
```

Comment:
I don't know why these `#{}` were added in the first place, it seems to work fine without!